### PR TITLE
docs: add interesting features

### PR DESCRIPTION
**Pull request by BoboTiG**
WDYT @lasconic?

- Is it worth adding such section?
- Do you see something missing?
- Improvement idea?

### CODE REVIEW

**Comment by BoboTiG** [comment]
Actually, screenhsots with math formulas are not SVGs... I will need to use up-to-date versions.
We should add 3 screenshots (math, chem, hiero).

Will do tomorrow.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,15 @@ Any help is welcome to improve those dictionaries:
 - :construction: [Norway](https://github.com/BoboTiG/ebook-reader-dict/releases/tag/no)
 - :construction: [Russian](https://github.com/BoboTiG/ebook-reader-dict/releases/tag/ru)
 
+## Interesting Features
+
+- [Kobo] When selecting a plural word, its singular form will be displayed.
+- [Kobo] When selecting a conjuged verb, its infinitive version will be displayed. → [raconteraient](docs/dict-raconteraient.png) • [dusse](docs/dict-dusse.png)
+- [Kobo] When a word is the plural form of another, and also a verb, all versions are displayed. → [grognasses](docs/dict-grognasses.png)
+- If a word contains several pronunciations, or genders, all are available. → [greffier](docs/dict-greffier.png)
+- Mathematic formulas, chemical foumlas, and hieroglyphs are converted to SVG. → [cercle unité](docs/dict-cercle_unite.png) • [hélicité](docs/dict-helicite.png)
```
Comment (lasconic) [inline README.md:33]: chemical **formulas**

File: README.md [hunk 2]
```diff
@@ -24,6 +24,15 @@ Any help is welcome to improve those dictionaries:
 - :construction: [Norway](https://github.com/BoboTiG/ebook-reader-dict/releases/tag/no)
 - :construction: [Russian](https://github.com/BoboTiG/ebook-reader-dict/releases/tag/ru)
 
+## Interesting Features
+
+- [Kobo] When selecting a plural word, its singular form will be displayed.
+- [Kobo] When selecting a conjuged verb, its infinitive version will be displayed. → [raconteraient](docs/dict-raconteraient.png) • [dusse](docs/dict-dusse.png)
```
Comment (lasconic) [inline README.md:30]: conjugated ?

File: README.md [hunk 3]
```diff
@@ -24,6 +24,15 @@ Any help is welcome to improve those dictionaries:
...
+- [Kobo] When a word is the plural form of another, and also a verb, all versions are displayed. → [grognasses](docs/dict-grognasses.png)
```
Comment (lasconic) [inline README.md:31]: and also a form of a verb ?

**Comment by lasconic** [comment]
Looks like a good addition but it's weird that the most advanced dictionary is the fr version and the documentation is in English. Should we create a README_fr ?

**Comment by lasconic** [comment]
Something like ? https://github.com/jonatasemidio/multilanguage-readme-pattern 
Not sure if there is support on github.

**Comment by BoboTiG** [comment]
Let's do english screenshots then :)

**Comment by BoboTiG** [comment]
We added variants for English, so it's already aligned with the French dict, right?

**Comment by BoboTiG** [comment]
I'm not sure about a FR README, we will need to add READMEs for all locales then.

**Comment by BoboTiG** [comment]
I'll find time the next week to introduce localized READMEs, and add newer screenshots.

**Comment by BoboTiG** [comment]
@lasconic I added something for the French dict, WDYT? If the format suits you too, I'll create other dicts.

File: docs/fr/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Dictionnaire Français Pour Votre Liseuse
+
+**eBook Reader Dictionaries** est un projet collaboratif utilisant l’immense base de connaissances du [Wiktionnaire](https://fr.wiktionary.org/) pour créer un dictionnaire complet, mis à jour quotidiennement.
+
+## Fonctionalités Intéressantes
+
+- [Kobo] Lors de la sélection d’un mot au pluriel, sa forme au singulier sera affichée.
```
Comment (BoboTiG) [inline docs/fr/README.md:7]: Should we keep `[Kobo]` for the 3 lines?

Comment (lasconic) [reply docs/fr/README.md:7]: Yes, it's true after all.

**Comment by BoboTiG** [comment]
I'm about to merge that one. I'll need your help for ES screenshots. About other locales, let's take screenshots when someone competent enough can give a hand (or if we find something in-between, let's open a specific PR).

**Pull request merged**